indigo/distribution.yaml

```diff
@@ -15062,6 +15062,21 @@ repositories:
       url: https://bitbucket.org/traclabs/trac_ik.git
       version: master
     status: developed
+  tracetools:
+    doc:
+      type: git
+      url: https://github.com/bosch-robotics-cr/tracetools.git
+      version: devel
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/bosch-robotics-cr/tracetools.git
```

Comment:
You should not use the same repository for the release.

Reply:
Yes, that will make a mess of your source repo.